tests/interface_call_forwarder/interface_forwarder.test.ts

```diff
@@ -1,295 +0,0 @@
-import { expect } from 'chai';
```

Comment:
Why did you remove the file?

Reply:
That's odd

Reply:
Problem was with this logic to delete generated files in the [folder](https://github.com/NethermindEth/warp/blob/97e70b0f48114e219f6ba9dfbf4947b3d1574950/tests/interfaceCallForwarder/interfaceForwarder.test.ts#L288)
Filename wasn't updated